Sources/TuistGenerator/Linter/TargetLinter.swift

```diff
@@ -188,29 +188,29 @@ class TargetLinter: TargetLinting {
     }
 
     private func lintDeploymentTarget(target: Target) -> [LintingIssue] {
```

Comment:
Shouldn't we add a linter that you haven't added the same deployment target more than once?
e.g.
```
deploymentTarget: [
  .iOS("11.0"),
  .iOS("13.0"),
]
```

shouldn't be allowed